ui/src/pages/frontsite/PatronProfile/PatronCurrentDocumentRequests/PatronCurrentDocumentRequests.js

```diff
@@ -63,10 +63,9 @@ export default class PatronCurrentDocumentRequests extends Component {
   renderNoResults = () => {
     return (
       <NoResultsMessage
-        messageHeader={'Requests for new literature'}
+        messageHeader={'No requests for new literature'}
         messageContent={
-          'You did not create any request for literature, ' +
-          'that is not available at the library.'
+          "You don't currently have any request for new literature not available in the catalog."
```

Comment:
nit: I would avoid apostrophes. It needs a comma possibly after literature. What about

"You do not have any request for new literature."